### DISCUSSION TITLE
BRIDGE_BookClub 5th Book

### DISCUSSION THREAD

**Discussion by fkdl0048** [BRIDGE-DEV/BRIDGE_BookClub#87] [Show and tell]
# BRIDGE_BookClub 5th Book

5회차에 진행할 서적을 미리 추천 받아서 결정하겠습니다.

현재 4회차까지 진행된 책은 다음과 같습니다.

- [1회차 클린코드](https://github.com/BRIDGE-DEV/BRIDGE_BookClub/issues?q=label%3ACleanCode+)
- [2회차 리팩터링](https://github.com/BRIDGE-DEV/BRIDGE_BookClub/issues?q=label%3ARefactoring+)
- [3회차 클린코더](https://github.com/BRIDGE-DEV/BRIDGE_BookClub/issues?q=label%3A%22The+Clean+Coder%22+)
- [4회차 객체지향의 사실과 오해](https://github.com/BRIDGE-DEV/BRIDGE_BookClub/issues?q=label%3A%22%EA%B0%9D%EC%B2%B4%EC%A7%80%ED%96%A5%EC%9D%98+%EC%82%AC%EC%8B%A4%EA%B3%BC+%EC%98%A4%ED%95%B4%22+)

항상 책을 해당 회차 진행원들과 회의를 통해 결정하기 때문에 이번 4회차에 참여하신 분들은 평소에 읽고 싶었거나 개인적 성장에 도움이 되는 책을 아래 코멘트로 적어주시면 됩니다.

*추천은 참여인원이 아니더라도 가능합니다.*

책을 선정하는 기준은 따로 없고 그때그때 달라지니 편하게 말씀해 주셔도 됩니다.

*개발 관련 or 게임 개발 서적으로 한정합니다.*

**Comment by fkdl0048** [comment]
- [클린 아키텍처](https://www.yes24.com/Product/Goods/77283734)
  - 소프트웨어 개발에 필요한
- [단위 테스트](https://www.yes24.com/Product/Goods/104084175)
  - 테스트코드에 대한 숙련도 및 이해
- [개발자의 글쓰기](https://www.yes24.com/Product/Goods/79378905)
  - 글쓰기 관련에 도움이 될 것 같은
- [실용주의 프로그래머](https://www.yes24.com/Product/Goods/107077663)
  - 개발자 마인드 셋

**Comment by joonyle99** [comment]
저도 '클린 아키텍처' 한표요..!
'실용주의 프로그래머'도 괜찮을거 같지만 소프트웨어 설계에 직접적으로 도움이 될거같은 클린 아키텍처가 좀 더 끌립니다 ;)

**Comment by luz0415** [comment]
- 클린 코드
- 클린 아키텍처
- 오브젝트
- 코드 CODE
- 디자인 패턴의 아름다움

다소 난이도가 있어보이는 책들이 있지만.. 한번 도전해보고 싶어서 추천해봅니다..!
